code/graph_algorithms/src/Bipartite_check/bipartite.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+#include <vector>
+#include <queue>
+int main()
+{
+    long int vertex , edge , u , v , c ;
+    bool flag=false;
```

Comment:
Could you format your code please. Please follow our C++ style guide, located in `guides/coding_style/C++/README.md` I believe.